src/components/SingleTopic.jsx

```diff
@@ -27,9 +31,9 @@ const SingleTopic = () => {
           {articles.map((article) => {
             return (
               <>
```

Comment:
With the `Each child in a list should have a unique "key" prop.` warning, this is appearing as although we have given the `<li>` tag a key attribute, the empty tag we have wrapped this li in `<> </>` needs a key. You can't give this a key attribute but this empty tag is shorthand so you can replace this with `<React.Fragment key={article.id}> </React.Fragment>` but as this is within a `<ul>` there may not be a need for the `<li></li>` to be wrapped with anything! :)